src/server.c

```diff
@@ -6445,7 +6448,8 @@ sds genRedisInfoString(const char *section) {
             "aof_last_cow_size:%zu\r\n"
             "module_fork_in_progress:%d\r\n"
             "module_fork_last_cow_size:%zu\r\n",
-            (int)server.loading,
+            (int)(server.loading && !server.async_loading),
```

Comment:
somehow i think it should not be added `!server.async_loading` here? (a bit confused?)
btw,  i think `async_loading` should also require a document, maybe missing,  i added it in https://github.com/redis/redis-doc/pull/1686

Reply:
It is indeed a bit confusing that in Redis both flags are set in this mode, but in info, it's either one or the other. 
But we concluded that for each one (users or developers) it makes sense to look at it this way.